### PR TITLE
Always use python3.6 for policy lambdas

### PR DESCRIPTION
**Pull request by whit537**
Alternative to #1546.

Custodian users write YAML, not Python. They shouldn't need to care what runtime we deploy to, right? Why not just always deploy to `python3.6`? We know that `c7n` and its deployment deps are portable between 2 and 3 (`ipaddress` is actually in 3's stdlib).

Mailer is a separate matter.

### CODE REVIEW

**Comment by whit537** [comment]
Rebased, was fb3c9388be55277c642e29ef3d4327febe3d79af.

File: c7n/ufuncs/s3crypt.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def get_function(session_factory, role, buckets=None, account_id=None):
         memory_size=256,
         timeout=30,
         role=role,
-        runtime="python2.7",
```
Comment (kapilt) [inline c7n/ufuncs/s3crypt.py]: not yet here either

File: c7n/mu.py [hunk 2]
```diff
@@ -624,7 +623,7 @@ class PolicyLambda(AbstractLambdaFunction):
     """Wraps a custodian policy to turn it into lambda function.
     """
     handler = "custodian_policy.run"
-    runtime = "python%d.%d" % sys.version_info[:2]
```
Comment (kapilt) [inline c7n/mu.py]: ugh.. no not yet. we're not even done with 3.6 compatibility. we're hyper focused on stability and compatibility (forward in this context), not pushing mass migrations.

**Comment by whit537** [comment]
> ugh.. no not yet. we're not even done with 3.6 compatibility. we're hyper focused on stability and compatibility

Okay, closing.

**Pull request closed without merging**